client/src/components/App/Content/SignUp/SignUpForm/useSignUpForm.ts

```diff
@@ -87,22 +119,52 @@ const useSignUp = ({ handleSaveUser }: useSignUpFormInCome) : useSignUpFormOutCo
             createUserLogger.error('create user was failed', Severity.CRITICAL);        
         })
         .finally(() => setIsLoading(false));
-    }
+    };
+
+    const editUser = (updatedUser: SignUpUser) => {
+        const updateUserLogger = logger.setup('Update user');
+        updateUserLogger.info('launching updateUser request', Severity.LOW);
+        axios.put('/users', {
+            ...updatedUser, 
+            languages : updatedUser.languages || [], 
+            city : updatedUser.city?.id, 
+            investigationGroup : updatedUser.investigationGroup?.id,
+            desk : updatedUser.desk?.id ? updatedUser.desk?.id : null,
```

Comment:
export to function along with lines 107-112

Reply:
👍🏼